GitTabExpansion.ps1

```diff
@@ -14,13 +14,17 @@ $subcommands = @{
     submodule = 'add status init update summary foreach sync'
     svn = 'init fetch clone rebase dcommit branch tag log blame find-rev set-tree create-ignore show-ignore mkdirs commit-diff info proplist propget show-externals gc reset'
     tfs = 'bootstrap checkin checkintool ct cleanup cleanup-workspaces clone diagnostics fetch help init pull quick-clone rcheckin shelve shelve-list unshelve verify'
-    flow = 'init feature release hotfix'
+    flow = 'init feature bugfix release hotfix support help version config finish delete publish rebase'

```

Comment:
Yes. I suppose technically 'help' isn't handled here, but even the [official completion](https://github.com/petervanderdoes/git-flow-completion) has it. Makes sense for consistency, if you think about it.